### ISSUE TITLE
Plugin marking included files as ignored

### ISSUE THREAD

**Issue by klevie** [hsz/idea-gitignore#344]
Since upgrading to the latest version of this (very useful!) plugin, I'm getting warnings that some files are ignored by git that actually aren't. 
This occurs when I use blocks like this in my .gitignore:

```
wp-content/plugins/*
!wp-content/plugins/plugin-to-include-anyway
!wp-content/plugins/another-exception
```

For now I have turned off the 'Inform about ignored files that are still tracked' and 'Inform about editing ignored file' options.
This is probably similar to issue #340, but I'm creating a separate issue since this is a different syntax that isn't understood properly.

**Comment by christian-weiss** [comment]
I can confirm that behavior with IntelliJ IDEA 2016.3.5 and plugin version 1.7.6

`build/releasePhar/
!build/releasePhar/.gitkeep`

.gitkeep file is marked as 'ignored, tracked with git' but in fact git do not ignore it because i told git to not ignore it. Message appears after update from IDEA 2016.3.2 to 2016.3.5 and updating plugin to 1..7.6.

**Comment by danielcompton** [comment]
I got the same problem too. It used to show up for any file with a `.com` in the directory path, as I had a `.com` rule in my global gitignore. e.g. `./resources/public/assets/css/fonts.googleapis.com/css-family-Roboto-Condensed.css` would show up as a problem. After removing `.com` from my global gitignore, I still get the warning, but there is nothing in the list to untrack:

<img width="529" alt="screenshot of intellij idea 13-03-17 11-59-23 am" src="https://cloud.githubusercontent.com/assets/811954/23836945/0ff94eb4-07e5-11e7-98be-1aa74a895854.png">

**Comment by almothafar** [comment]
Yes I just about reported the same issue, even with the following line:
```
/project
!/project/build.properties
!/project/plugins.sbt
```

The plugin show me the message 

> Ignored files found that are tracked: .ignore plugin found some files that are tracked with Git which are also listed in .gitignore file. 

And advice me to remove them with git! oh no, don't do that! it is not a joke!

**Comment by AndersDJohnson** [comment]
Same issue here with generally excluded directories then a `!.gitkeep` at the end. Any workarounds anyone found?

**Comment by AndersDJohnson** [comment]
Related to #310?

**Comment by hsz** [comment]
@klevie @christian-weiss @danielcompton @almothafar @AndersDJohnson 
Thanks for this report!

This issue has been resolved - commit is attached to #350.
Fix will be released soon with v2.0.0 version.

CC @cjbarth @indivisible-irl
